packages/heptagon/heptagon.1.03.00/opam

```diff
@@ -9,10 +9,13 @@ build: [
 ]
 tags: [ "flags:light-uninstall" ]
 depends: [
-  "ocamlfind"
-  "menhir" {>= "20141215"}
-  "ocamlgraph"
-  "camlp4"
+  "ocamlfind"  {build}
+  "menhir"     {build & >= "20141215"}
+  "ocamlgraph" {build}
+  "camlp4"     {build}
   "ocamlbuild" {build}
 ]
-depopts: ["lablgtk"]
+depopts: [
+  "lablgtk"
+  "reatk"
```

Comment:
This `reatk` dependency is not present in the stable OPAM repository. Do you mind if I remove it from the central repository?